ci/docker/runtime_functions.sh

```diff
@@ -609,14 +614,17 @@ build_ubuntu_gpu_cmake_mkldnn() {
     cd /work/build
     cmake \
         -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-        -DCMAKE_C_COMPILER_LAUNCHER=ccache \
-        -DENABLE_TESTCOVERAGE=ON   \
-        -DUSE_CUDA=1               \
-        -DUSE_CUDNN=1              \
-        -DUSE_MKLML_MKL=1          \
-        -DUSE_MKLDNN=1             \
-        -DCMAKE_BUILD_TYPE=Release \
-        -G Ninja                   \
+        -DCMAKE_C_COMPILER_LAUNCHER=ccache   \
+        -DENABLE_TESTCOVERAGE=ON             \
+        -DUSE_CUDA=1                         \
+        -DUSE_CUDNN=1                        \
+        -DUSE_MKLML_MKL=1                    \
+        -DUSE_MKLDNN=1                       \
+        -DCMAKE_BUILD_TYPE=Release           \
+        -DCUDA_ARCH_NAME=Manual              \
+        -DCUDA_ARCH_BIN=52,70                \
+        -DCUDA_ARCH_PTX=52                   \
```

Comment:
Could we extract these ones as well? 

Also, is the PTX the minimum compatible version or what does it exactly mean? Sorry if it's a trivial question

Reply:
Not a trivial question.  Actually I want to experiment a bit here to see how these two options correspond to arch params to nvcc.  PTX is usually the intermediate code that can be jitted to cubin.  So you're completely correct, it usually acts as a minimum compute capability for CUDA code.  In theory CI should only need BINs so I'll see what happens if we get rid of it.